src/main/features/core/tray.js

```diff
@@ -29,6 +29,18 @@ function getAppIconFileName() {
   return path.resolve(currentIconPath, appIconFileName);
 }
 
+// Trim the song title to fit in the menu
```

Comment:
Although in general comments are nice, if it's obvious what the function does they just clutter the code IMO